### DISCUSSION TITLE
I would like to know if TRON Network is supported，Thanks

### DISCUSSION THREAD

**Discussion by LukeAntarctic** [rainbow-me/rainbowkit#2166] [Q&A] [answered]
There is a demand now to connect to Tron, Can you provide me with some help

**Comment by DanielSinclair** [comment, marked as answer]
Yes, I believe Tron should be compatible now. Make sure to upgrade to the latest versions of Wagmi and Viem and import the `tron` chain with chain id `728126428`. Let me know if you find any incompatibilities!

**Reply by Cmonorail** [reply]
Hello can you send me detail? I can not find it.thanks very much

**Reply by DanielSinclair** [reply]
@Cmonorail You reference the chain implementation in Viem: https://github.com/wevm/viem/blob/main/src/chains/definitions/tron.ts

**Reply by Cmonorail** [reply]
> @Cmonorail You reference the chain implementation in Viem: https://github.com/wevm/viem/blob/main/src/chains/definitions/tron.ts

Thanks Wagmi also supported this?

**Reply by DanielSinclair** [reply]
Yes
